docs/caching/managing-cache.md

```diff
@@ -62,3 +62,15 @@ If you are using a self-hosted remote runner, you can configure the cache policy
 The command `earthly prune` will work on remote runners too, albeit without the `--reset` flag, which is not supported in a remote setting.
 
 To cause a satellite to restart with a fresh cache, you can use the command `earthly sat update --drop-cache`.
+
+## Auto-skip cache
+
+The auto-skip cache is a cache that is used to skip large parts of a build in certain situations. It is used by the `earthly --auto-skip` and `BUILD --auto-skip` commands.
+
+Unlike the layer cache and the cache mounts, the auto-skip cache is global and is stored in a cloud database.
+
+To clear the entire auto-skip cache for your Earthly org, you can use the command `earthly prune-auto-skip`.
```

Comment:
Does `prune-auto-skip` exist yet? Placeholder?

Reply:
It does not - taking a leap of faith here lol. Let me know if you have ideas for a different name.